src/test/java/com/dsrm/dsrmbackend/UserControllerIntegrationTests.java

```diff
@@ -73,4 +78,36 @@ void retrieveUsersInRange() throws Exception {
                 .andExpect(jsonPath("$.content[1].surname").value("Nowak"))
                 .andExpect(jsonPath("$.content[1].email").value("test02@wp.pl"));
     }
+
+    @Test
+    void validPatchExistingUser() throws Exception {
+        ObjectMapper objectMapper = new ObjectMapper();
+        UserRolesOnlyDTO userRolesOnlyDTO  = new UserRolesOnlyDTO();
+        List<Long> longs = new ArrayList<>();
+        longs.add(1L);
+        userRolesOnlyDTO.setRoles(longs);
+        this.mockMvc.perform(patch("/users/1/roles").contentType(MediaType.APPLICATION_JSON)
+                .content(objectMapper.writeValueAsString(userRolesOnlyDTO))
+                ).andExpect(jsonPath("$.email", equalTo("test01@wp.pl")))
+                .andExpect(jsonPath("$.name", equalTo("Jan")))
```

Comment:
We want to return void so we won;t check this here. I would autowire userRepo and fetch user using repository and check if the role were added, because it is only information which we are interested in.